AppController/djinn.rb

```diff
@@ -4507,12 +4507,20 @@ def stop_ejabberd
   def create_appscale_user
     uac = UserAppClient.new(my_node.private_ip, @@secret)
     password = SecureRandom.base64
+
     begin
+      retries ||= 0
```

Comment:
There is already a RETRY constant defined. Perhaps you could use something like RETRY.downto(0) {}?